pandas/tests/test_generic.py

```diff
@@ -996,6 +996,15 @@ def test_describe_percentiles_insert_median(self):
         self.assertTrue('0%' in d1.index)
         self.assertTrue('100%' in d2.index)
 
+    def test_describe_percentiles_insert_median_ndarray(self):
+        df = tm.makeDataFrame()
+        try:
```

Comment:
you don't need the try/except at all; nose will simply fail the test if this raises.

rename d1 -> result, d2 -> expected